### PR TITLE
implement train arrival log function

### LINKED ISSUE #168: Train arrival Log function
## User stories
As a developer I want to have the train arrival function in order to notify the logger about a arriving train at a station.

## Dependencies
Depends on: 
 - #14 

## (Wiki) References
- https://github.com/BP2022-AP1/bp2022-ap1/wiki/Logging

## Acceptance Criteria
- [x] implement train arrival function

### PR DESCRIPTION
**Pull request by LucasDerReisende**
Fixes #168 

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [x] Breaking changes anounced
- [x] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [x] Another dev reviewed and approved

### CODE REVIEW

File: src/logger/logger.py [hunk 1]
```diff
@@ -62,7 +66,14 @@ def arrival_train(self, tick: int, train_id: str, station_id: str) -> Type[None]
         :param station_id: The id of the station
         :rtype: None
         """
-        pass  # not implemented yet # pylint: disable=W0107
+        TrainArrivalLogEntry.create(
+            timestamp=datetime.now(),
+            tick=tick,
+            message=f"Train with ID {train_id} arrived at station with ID {station_id}",
+            run_id=self.run_id,
```
Comment (antonykamp) [inline src/logger/logger.py:73]: I'm unsure if the other components forward the `run_id` to the logger or if the logger instance receives the `run_id` during the initialization. Would you happen to know if we already talked about it? 🤔

Comment (LucasDerReisende) [reply src/logger/logger.py:73]: I'm not quite sure when we talked about it or how we made this decision, but I came to the result, that we should pass the run (or run_id) when initializing
If you see any problem with this or discuss it further, please just let me know :)

Comment (antonykamp) [reply src/logger/logger.py:73]: > but I came to the result, that we should pass the run (or run_id) when initializing

Sounds great 👍 🥳

**Review by antonykamp — APPROVED** [review]
Keep rocking again 🤖 But don't forget to update the documentation
 https://github.com/BP2022-AP1/bp2022-ap1/wiki/Architecture#logger

**Pull request merged**